python/tvm/autotvm/measure/measure_methods.py

```diff
@@ -213,12 +208,13 @@ def _fbuild(inp):
         """ Local build function."""
         func, args = _build_func(inp, build_option, kwargs)
 
+        tmp_dir = util.tempdir()
         if not kwargs.get('use_ndk', False):
-            file_name = "tmp_func_%0x.tar" % getrandbits(64)
```

Comment:
We no longer need the filename randomization that we had before? I wonder if this will still be useful if remote sessions are reused and overwritten files are not handled properly (e.g., Android).